utils/src/test/java/org/web3j/utils/StringsTest.java

```diff
@@ -19,16 +21,11 @@
 
     @Test
     public void testToCsv() {
-        assertThat(toCsv(Collections.<String>emptyList()), is(""));
+        assertThat(toCsv(Collections.emptyList()), is(""));
```

Comment:
The parameterized value is so that this function works with Java 1.6 in the Android branch.

Reply:
Android.  Ah-ha.  Maybe it's better then to change the gradle file to require Java v 1.6 ?  (Unless I missed it, I may have only checked the top-level config and not the module's config.)

Reply:
Do we want to change these lines in the root build.gradle file:

    sourceCompatibility = 1.8
    targetCompatibility = 1.8